b2/console_tool.py

```diff
@@ -258,6 +286,24 @@ def test_raw_api(self, args):
 
     # bucket
 
+    def cancel_all_unfinished_large_files(self, args):
```

Comment:
I can argue that both ways.  I guess the question is whether other developers using the library will want that functionality, and the answer is probably yes.